app/components/Search/mazemapAutocomplete.tsx

```diff
@@ -87,6 +87,7 @@ function mazemapAutocomplete<Props>({
                 ),
               });
             })
+            .catch((error) => {})
```

Comment:
What's the point of this? You don't do anything with the error?

Reply:
No, but something seems to have changed about the search in mazemap so it will throw something like "search cancelled because a new one was initiated" when writing (because that triggers new searches)